Measure-radial-velocities/avg_rv.py

```diff
@@ -0,0 +1,34 @@
+import pandas as pd
+import numpy as np
+import weighted_stddev
+
+
+def avg_rv(path_to_df):
+    """Calculates a weighted average and uncertainty of the radial velocities in a given dataframe
+
+    Input: path to pandas dataframe with columns 'obj_rv' and 'obj_unc'
```

Comment:
Formally, the 'input' of a function declaration is called an 'argument'. There really arent any enforced styles but this tends to be the convention for python coders/developers